Source/WebKit/WebProcess/WebCoreSupport/SessionStateConversion.cpp

```diff
@@ -130,14 +130,14 @@ static Ref<FormData> toFormData(const HTTPBody& httpBody)
         }, [&] (const HTTPBody::Element::FileData& data) {
             formData->appendFileRange(data.filePath, data.fileStart, data.fileLength.value_or(BlobDataItem::toEndOfFile), data.expectedFileModificationTime);
         }, [&] (const String& blobURLString) {
-            formData->appendBlob(URL { blobURLString });
+            formData->appendBlob(URL { blobURLString }, SecurityOrigin::createFromString(mainFrameURL)->data(), partitionBlobRegistry);
```

Comment:
Since we don't really need a SecurityOrigin, seems we could use `SecurityOriginData::fromURL(URL(mainFrameURL))`